src/org/opendatakit/briefcase/ui/export/components/FormsTableViewModel.java

```diff
@@ -101,10 +102,18 @@ private JButton buildOverrideConfButton(FormStatus form) {
       );
       dialog.onRemove(() -> removeConfiguration(form));
       dialog.onOK(configuration -> {
-        if (configuration.isEmpty())
+        if (configuration.isEmpty() &&
+                (((BriefcaseFormDefinition) form.getFormDefinition()).isFileEncryptedForm() ||
```

Comment:
The configuration should already know if it belongs to a form that needs a `PrivateKey` or not. This shouldn't be needed here.